app/controllers/concerns/votes.rb

```diff
@@ -0,0 +1,37 @@
+module Votes
+  extend ActiveSupport::Concern
+
+  def votes
```

Comment:
create a votes_manager and try service layer architecture

also where is the logic for total count?

Reply:
> create a votes_manager and try service layer architecture

+1

> also where is the logic for total count?

I thought it was to be implemented using sidekiq